src/EventStore.LogV3/Raw.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using EventStore.LogCommon;
+
+namespace EventStore.LogV3 {
+	// todo: alignment, padding
+	public static class Raw {
+		[StructLayout(LayoutKind.Explicit, Size = Size, Pack = 1)]
+		public struct RecordHeader {
+			[FieldOffset(0)] private LogRecordType _type;
+			[FieldOffset(1)] private byte _version;
+			// todo: too much padding for somethng that will occur every record
+			[FieldOffset(8)] private long _ticks;
+			[FieldOffset(16)] private Guid _recordId;
+
+			// todo: remove this if possible
+			[FieldOffset(48)] private long _logPosition;
```

Comment:
```suggestion
			[FieldOffset(32)] private long _logPosition;
```

@timothycoleman Is there a specific reason why this field starts at offset 48 instead of 32? (since Guid is 128-bit/16 bytes)

Reply:
good spot, there was probably another field before it that got removed :D

Reply:
great, thanks @timothycoleman . As discussed, the structs will be finalized at a later stage.